src/main/java/reposense/model/AuthorConfiguration.java

```diff
@@ -145,10 +145,21 @@ public void addAuthor(Author author, List<String> ignoreGlobList) {
     public void removeIgnoredAuthors(List<String> ignoredAuthorsList) {
         for (String author : ignoredAuthorsList) {
             if (authorEmailsAndAliasesMap.containsKey(author)) {
-                authorList.remove(authorEmailsAndAliasesMap.get(author));
+                Author ignoredAuthor = authorEmailsAndAliasesMap.get(author);
+                List<String> aliases = new ArrayList<>(ignoredAuthor.getAuthorAliases());
+                removeIgnoredAuthorsInformation(author, aliases, ignoredAuthor);
             }
         }
-        resetAuthorInformation();
+    }
+
+    /**
+     * Removes information of the {@code author} along with its {@code aliases}
+     */
+    public void removeIgnoredAuthorsInformation(String author, List<String> aliases, Author ignoredAuthor) {
+        authorList.remove(ignoredAuthor);
+        authorDisplayNameMap.remove(ignoredAuthor);
+        authorEmailsAndAliasesMap.remove(author);
+        aliases.forEach(alias -> authorEmailsAndAliasesMap.remove(alias, ignoredAuthor));
```

Comment:
Line 161 and 162 use different variants of the `remove` method
Both should be suitable, but I think `Map.remove(Object key)` is simpler here. What do you think?

Reply:
So, the thing is - Author, alias, email all are set as `keys`. So we need to remove them all. 
As pointed by you, email has been missed out. So I will include that. 

We can use `Map.remove(Object key)` as we don't have duplicates anyway.